example-viewbinding/src/main/java/com/xwray/groupie/example/viewbinding/CarouselGroup.kt

```diff
@@ -0,0 +1,59 @@
+package com.xwray.groupie.example.viewbinding
+
+import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.RecyclerView.AdapterDataObserver
+import androidx.recyclerview.widget.RecyclerView.ItemDecoration
+import com.xwray.groupie.*
+import com.xwray.groupie.example.viewbinding.item.CarouselItem
+
+/**
+ * A group that contains a single carousel item and is empty when the carousel is empty
+ */
+class CarouselGroup(
+    itemDecoration: ItemDecoration,
+    adapter: GroupAdapter<GroupieViewHolder>
+) : Group {
+
+  private var isEmpty = true
+  private val adapter: RecyclerView.Adapter<*>
+  private var groupDataObserver: GroupDataObserver? = null
+  private val carouselItem: CarouselItem
+
+  init {
+    this.adapter = adapter
```

Comment:
Can you please reformat this whole example project to use an indentation of 4 to keep it in line with the other ones?